pkg/network/events/monitor_windows.go

```diff
@@ -0,0 +1,79 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package events handles process events
+package events
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	"go4.org/intern"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil/iisconfig"
+)
+
+func getProcessStartTime(ev *model.Event) time.Time {
+	if ev.GetEventType() == model.ExecEventType {
+		return ev.GetProcessExecTime()
+	}
+	// windows does not have a fork event.
+	return time.Time{}
```

Comment:
Similar to above comment in the Linux variant of this function - is it fine to return `time.Time{}` or should we be logging a warning / error and return it that indicates we got an `EventType` we weren't expecting?